workflow-templates/sphinx.yml

```diff
@@ -0,0 +1,38 @@
+# Builds the Sphinx documentation in the specified folder.
+# Default values are docs/ for the directory where the Sphinx
+# documentation is located and true to run linkcheck
+# To pass a new folder, set directory_path to the folder path
+# To disable linkcheck, set check_link to any value 
+
+---
+name: sphinx
+
+on: 
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env: 
+      directory_path:
+      check_link: true # set to false to remove linkcheck
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Sphinx install
+        run: pip install --upgrade sphinx
+      - name: Set directory
+        run:  |
+          VAR=${{ env.directory_path }}
+          echo "docs_dir="${VAR:-./docs} >> $GITHUB_ENV
+      - name: Build doc
+        run: |
+          cd $docs_dir
+          if [ "${{ env.check_link }}" = true ]; then
+            make clean html linkcheck
```

Comment:
Another option here would be to have added an argument "$addition_targets"

Reply:
actually we could have targets at the top similar to what I did for https://github.com/ome/.github/pull/15
``clean html linkcheck`` being the default.
and one could change that depends on the requirement.
that is probably easier to use if we, for example, want to add pdf